src/ServiceInsight/ServiceControl/DefaultServiceControl.cs

```diff
@@ -707,7 +711,7 @@ void LogResponse(IRestResponse response)
         void LogError(IRestResponse response)
         {
             var exception = response?.ErrorException;
-            var errorMessage = response != null ? string.Format("Error executing the request: {0}, Status code is {1}", response.ErrorMessage, response.StatusCode) : "No response was received.";
+            var errorMessage = response != null ? string.Format("Error executing the request: {0}, Status code is {1}, content: {2}", response.ErrorMessage, response.StatusCode, response.Content) : "No response was received.";
```

Comment:
Not sure it compiles, I wrote it in the browser. This reads better than the string format positional arguments.

```suggestion
            var errorMessage = response != null ? $"Error executing the request: {response.ErrorMessage}, Status code is {response.StatusCode}, content: {response.Content}" : "No response was received.";
```

Reply:
I changed it in Rider, as I want to make sure it compiles ;)